pkg/virt-controller/watch/vmi_test.go

```diff
@@ -3144,7 +3144,7 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 			})
 		}
 
-		fakeHotPlugRequest := func(vmi *virtv1.VirtualMachineInstance, addOpts []virtv1.AddInterfaceOptions) {
+		fakeHotPlugRequest := func(vmi *virtv1.VirtualMachineInstance, addOpts []AddInterfaceOptions) {
```

Comment:
This helper needs to be replaced with one that creates a new network, i.e. `newNetwork(name)`.
I refactored this mess as part of a different PR, so either you copy from there or we will keep it as is and assure the other will get in after.

The current usage of this odd `addInterfaceOptions` is very confusing.

Reply:
If you already did it, I prefer not to copy paste it. Let's merge your PR after this one,